src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -419,7 +419,7 @@ public void Initialize()
                     }));
 
                 _eventSubscriptions.Add(EventManager.OfType<HostRestartEvent>()
-                    .Subscribe((msg) => ScheduleRestartAsync(false)));
+                    .Subscribe((msg) => ScheduleRestartAsync(false).ContinueWith(t => _startupLogger.LogError(t.Exception.Message), TaskContinuationOptions.OnlyOnFaulted)));
```

Comment:
@fabiocav this is the only non-styling impacting changed, besides the gRPC build change

Reply:
1. Look for an overload (or extension) that logs the exception... we don't want to lose the stack.
2. Continue this synchronously.

Reply:
e.g. https://github.com/fabiocav/azure-webjobs-sdk-script/blob/116b3e7703e0a806c498b9948e511b4b922b5d61/src/WebJobs.Script/Description/DotNet/DotNetFunctionInvoker.cs#L218